products/uosv20/product.yml

```diff
@@ -0,0 +1,27 @@
+product: uosv20
+full_name: Uniontech OS V20
+type: platform
+
+benchmark_id: UOS-V20 
+benchmark_root: "../../linux_os/guide"
+
+profiles_root: "./profiles"
+
+pkg_manager: "yum"
```

Comment:
Please confirm that UOS uses the `yum` package manager, as my quick research shows that UOS is based on Debian.

Reply:
UOS is based on centos8 ,so,it should be .yum